### PR TITLE
Vls export

### PR DESCRIPTION
**Pull request by alexanderGerbik**
Allow to use "validate" and "export to grey" features for vls tasks.

Address some technical debt.

### CODE REVIEW

File: cvat/apps/engine/ddln/tasks/__init__.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+def create_task_handler(task_type):
+    if task_type == "spotter":
+        return SpotterTaskHandler()
+    if task_type == "vls":
+        return VlsTaskHandler()
+    raise ValueError("Unexpected task type: {!r}".format(task_type))
+
+
+def guess_task_type(task):
```
Comment (gyakushev) [inline cvat/apps/engine/ddln/tasks/__init__.py:13]: why is this necessary?

**Review by gyakushev — APPROVED** [review]
Looks like a good refactoring.
Thanks!

File: cvat/apps/engine/ddln/tasks/__init__.py (hunk 1 above)
Comment (alexanderGerbik) [reply cvat/apps/engine/ddln/tasks/__init__.py:13]: Task type is identified by the labels which were added to the task. it should almost always properly detect task type, but it might fail if wrong labels were attached on task creation.

Identified task type is then used to run right logic on task validation or task export.
